### PR TITLE
WRO-9302: [Scroller]Should not scroll when focus moves paging control to paging control

### PR DESCRIPTION
**Pull request by vladut-clapou-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
This PR solves a bug on the Scroller component. When focusableScrollbar=true, focusing the bottom button of the vertical scroller and then moving the focus to the right, beside changing the focus to the right button of the horizontal scroller, triggered also a scroll towards the button of the horizontal scroller.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-9302

### Comments
Enact-DCO-1.0-Signed-off-by: Clapou Vladut [vladut.clapou@lgepartner.com](mailto:vladut.clapou@lgepartner.com)

### CODE REVIEW

**Comment by daniel-stoian-lgp** [comment]
please add changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/MediaPlayer` layout issues for Cobalt, Carbon, Copper, Electro, Titanium skins
 - `agate/Picker` and `agate/RangePicker` to match latest design for Silicon skin
 - `agate/Scroller` to update scrollButtons state on initial render
+- `agate/Scroller` shoould not scroll when focus moves from a scroll button to another
```
Comment (daniel-stoian-lgp) [inline CHANGELOG.md:13]:
```suggestion
- `agate/Scroller` should not scroll when focus moves from a scroll button to another
```

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Comment by seunghoh** [comment]
@vladut-clapou-lgp Would you please update the PR description? It doesn't have any information. For reviewer or merger, what do you want to say? Please read the PR template and put the information
[//]: # (What is the impact of this change and *why* was it made?)
CC @daniel-stoian-lgp

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**